analyzers/conversion_overflow.go

```diff
@@ -224,3 +291,71 @@ func isIntOverflow(src string, dst string) bool {
 
 	return false
 }
+
+func isRelevantBinOp(binOp *ssa.BinOp, x ssa.Value) bool {
```

Comment:
Maybe rename this something like `isBoundCheck` to match other naming?

Reply:
resolved